SwiftWisdomTests/Rx/OperatorTests.swift

```diff
@@ -8,16 +8,17 @@
 
 import XCTest
 import RxSwift
+import RxCocoa
 @testable import SwiftWisdom
 
 
 class OperatorTests: XCTestCase {
 
     func testAddCompositeDisposable() {
         let compositeDisposable = CompositeDisposable()
-        let observable = Observable<String>.never()
+        let observable = Observable<String?>.never()
         XCTAssertEqual(0, compositeDisposable.count)
-        UILabel().rx_text <- observable >>> compositeDisposable
+        let _ = UILabel().rx.text <- observable >>> compositeDisposable
```

Comment:
The use case of a CompositeDisposable is slightly different than a DisposeBag.  The former you manage the lifecycle of manually and tell it to dispose when you are done with it.  The latter you typically just put a bunch of disposables into and forget about them.

Why would you want to remove a disposable from a CompositeDisposable?  I'm not entirely sure of the use case there, but the API seems to support it.  Maybe you'll have a subscription that would change as a result of something and you don't want it to be grouped anymore.

Do you think it would be better to drop this functionality for compression?